src/SDK.ts

```diff
@@ -182,6 +182,7 @@ export default class SDK implements ISDK {
 
     let requestHeaders = {};
     addOcUserAgentHeader(this.ocUserAgent, requestHeaders);
```

Comment:
Can this be removed since `setCorrelationIdInHeader()` is encapsulating the logic?